blocks/video/video.js

```diff
@@ -104,42 +97,7 @@ const decorateVideoLink = (link, label = 'Play') => {
   return playBtn;
 };
 
-/**
- * Create a new YT Player and store the result of its player ready event.
- * @param element iFrame element YouTube player will be attached to.
- * @param videoId The YouTube video id
- */
-const loadYouTubePlayer = (element, videoId) => {
-  const onPlayerReady = (event) => {
-    playerMap[videoId] = event.target;
-  };
-
-  // Create a new iframe element
-  const iframe = document.createElement('iframe');
-  iframe.src = `https://www.youtube.com/embed/${videoId}`;
-
-  // Add the data-hj-allow-iframe attribute
-  iframe.setAttribute('data-hj-allow-iframe', '');
```

Comment:
This code should not be removed. We need to add the `data-hj-allow-iframe` to the YT iframe.